### PR TITLE
Eliminate dead architecture tags

### PR DESCRIPTION
**Pull request by wrongnull**
see https://github.com/ziglang/zig/pull/20708
https://github.com/ziglang/zig/pull/20402

### CODE REVIEW

File: src/aro/target.zig [hunk 1]
```diff
@@ -685,9 +641,8 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
         .dragonfly => "dragonfly",
         .freebsd => "freebsd",
         .fuchsia => "fuchsia",
-        .kfreebsd => "kfreebsd",
         .linux => "linux",
-        .lv2 => "lv2",
+        .ps3 => "ps3",
```
Comment (Vexu) [inline src/aro/target.zig:645]: The LLVM name shouldn't be changed.

Comment (wrongnull) [reply src/aro/target.zig:645]: Why so? It has been changed here https://github.com/ziglang/zig/pull/20708/files#diff-73b15ac9aed4b9aca85583d8caa883b2a5211a63b411ae6ef088a6d8b92156f6L671

Comment (wrongnull) [reply src/aro/target.zig:645]: not sure what did you mean

Comment (wrongnull) [reply src/aro/target.zig]: Oh, I'm sorry. Done

**Pull request merged**